classes/03class/exercises/c03-docker02/starkmatt/Dockerfile.v2

```diff
@@ -0,0 +1,2 @@
+FROM httpd:2.4.41-alpine
+EXPOSE 80
```

Comment:
I reckon the `/usr/local/apache2/htdocs/` folder is not empty with this Dockerfile...
```
Named exercise02 with a tag v2 with the following pre-requisites
Dockerfile must be named Dockerfile.v2
Final image is based from httpd:2.4.41-alpine
The folder /usr/local/apache2/htdocs/ must be empty
```

Reply:
You are right!
Would we run the CMD rmdir -p /usr/local/apache2/htdocs/ in the Dockerbuild file ?

Reply:
you need to run a RUN instruction to execute the removal of the files during the image creation.